docs/javascript/typescript.md

```diff
@@ -0,0 +1,173 @@
+# TypeScript
+
+## Consuming WoltLab Suite’s Types
+
+To consume the types of WoltLab Suite you will need to install the [WoltLab/WCF](https://github.com/WoltLab/WCF) using npm using a git URL that refers to the appropriate branch.
```

Comment:
```suggestion
To consume the types of WoltLab Suite, you will need to install the [WoltLab/WCF](https://github.com/WoltLab/WCF) using npm using a git URL that refers to the appropriate branch.
```

"… install the WoltLab/WCF *package*"?

Also using "using" twice: May-be "via npm using …"?

Reply:
I've pushed a fix. PTAL.